metadata.json

```diff
@@ -15,6 +15,13 @@
     "gunicorn"
   ],
   "operatingsystem_support": [
+     {
+      "operatingsystem": "AIX",
+      "operatingsystemrelease": [
+        "6100",
```

Comment:
`facter operatingsystemrelease` returns `6100` instead of `6.1` ?

Reply:
{name => AIX, family => AIX, release => {major => 6100-09-11-1810, full => 6100-09-11-1810}

Reply:
I would keep this with family only and then handle the releases.

Reply:
The return from operatingsystem release is 6100-09-11-1810..

Reply:
hrm this is pretty crappy from facter :(
@ghoneycutt do you think we should keep the current version of the metadata or update it to the actual value from facter?

Reply:
I see for gentoo we don't care for release as well..

Reply:
We don't care for the versions on Gentoo and Archlinux because those are rolling release distributions. For everything else we have to care about it. Based on this data, the facts are mocked during the tests (and some other stuff).